### ISSUE TITLE
[Onsend Addin] Unable to provide option in manifest to load Outlook Web addin only on macos platform

### ISSUE THREAD

**Issue by Premsagar-lab** [OfficeDev/office-js#1882]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

Our web addin is currently designed to work on Macos only. Addin is deployed using Office 365 CDS for user. Addin is loaded automatically for that user even on windows. So the addin is invoked even when they are working on windows platform which adds to delay in sending email.

We dont have option to specify platform on which web addin should work or deployed.

In Manifest file, we can define addin to be loaded with DesktopFormFactor element. But this does not provide any option to restrict addin from getting loaded on specific platform.  

## Expected Behavior

1. A manifest element should be available to provide platform where addin will work.
2. CDS should provide option to administrator to select platform.

## Current Behavior

Addin loads explicitly for desktop users independent of platform.

## Steps to Reproduce, or Live Example

* Link to live example: ______

* Additional details:
  1. ______
  2. ______
  3. ______

## Context

Windows users are facing delay in sending email as the addin is loaded through network and invoked. It returns only after platform check.

**Comment by Premsagar-lab** [comment]
@exextoc Any update on this?

**Comment by exextoc** [comment]
Currently, option to restrict addin from getting loaded on specific platform is not a part of the product. If manifest has DesktopFormFactor, add-in will load on Win32, Mac and OWA. And if it contains MobileFormFactor, add-in will load on iOS and Android. We track Outlook add-in feature requests on our  [Tech Community Page](https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/idb-p/Microsoft365DeveloperPlatform). Please submit your request there and choose the appropriate label(s). Feature requests on Tech Community are considered when we go through our planning process.

**Comment by SWei1234** [comment]
Hi @exextoc 
Also have this feature request. Wish this will have a timeline on future.
Thanks
